sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/_base.py

```diff
@@ -219,14 +238,25 @@ def _transmit(self, envelopes: List[TelemetryItem]) -> ExportResult:
                         _update_requests_map(_REQ_SUCCESS_NAME[1], 1)
                     reach_ingestion = True
                     result = ExportResult.SUCCESS
+
+                    # Track successful items in customer statsbeat
+                    if self._customer_statsbeat_metrics and self._should_collect_customer_statsbeat():
+                        _track_successful_items(self._customer_statsbeat_metrics, envelopes)
                 else:  # 206
                     reach_ingestion = True
                     resend_envelopes = []
                     for error in track_response.errors:
                         if _is_retryable_code(error.status_code):
                             resend_envelopes.append(envelopes[error.index])  # type: ignore
+                            # Track retried items in customer statsbeat
+                            if self._customer_statsbeat_metrics and self._should_collect_customer_statsbeat():
+                                _track_retry_items(self._customer_statsbeat_metrics, resend_envelopes, error)
                         else:
                             if not self._is_stats_exporter():
+                                # Track dropped items in customer statsbeat, non-retryable scenario
+                                if self._customer_statsbeat_metrics and self._should_collect_customer_statsbeat():
+                                    if error is not None and hasattr(error, "index") and error.index is not None:
+                                        _track_dropped_items(self._customer_statsbeat_metrics, [envelopes[error.index]], error.status_code)
```

Comment:
Nit: You can create a list of dropped items and then simply call _track_dropped_items once at the end without having to call it multiple times with the a list of one item.